testing/chain.go

```diff
@@ -457,7 +464,12 @@ func (chain *TestChain) CreateTMClientHeader(chainID string, blockHeight int64,
 	blockID := MakeBlockID(hhash, 3, tmhash.Sum([]byte("part_set")))
 	voteSet := tmtypes.NewVoteSet(chainID, blockHeight, 1, tmproto.PrecommitType, tmValSet)
 
-	commit, err := tmtypes.MakeCommit(blockID, blockHeight, 1, voteSet, signers, timestamp)
+	var signerArr []tmtypes.PrivValidator
+	for _, v := range tmValSet.Validators {
```

Comment:
is it going to be problematic that you iterate over a map which may not give correct ordering of the validators?

Reply:
No the validators is not a map. It is an ordered array.

We then have an unordered map of signers which we convert into an array of signers that is ordered the same way as validators array

Reply:
Can you add a comment explaining this?